otherlibs/unix/getgr.c

```diff
@@ -18,10 +18,13 @@
 #include <caml/alloc.h>
 #include <caml/memory.h>
 #include "caml/unixsupport.h"
-#include <errno.h>
-#include <stdio.h>
+#include <unistd.h>
 #include <grp.h>
 
+#if !defined(HAVE_GETGRNAM_R) || !defined(HAVE_GETGRID_R)
+#include <errno.h>
+#endif
```

Comment:
That's wrong. The `_r` codepath does not refer to `errno` but nevertheless uses `ERANGE` which comes from `errno.h`, so you should always include that file instead of relying upon it being included by another header.

```suggestion
#include <errno.h>
```

Reply:
Right, fixed.